JSDemos/Demos/TreeList/RowEditing/description.md

```diff
@@ -1 +1,16 @@
-This example demonstrates editing in _&laquo;row&raquo;_ mode. Each row is&nbsp;supplied with an _&laquo;Edit&raquo;_ link, a&nbsp;click on&nbsp;which switches the row into the editing state. Editors vary from one cell to&nbsp;another: they may be&nbsp;calendars, select boxes, simple text boxes, etc. Note that you can also add or&nbsp;delete a&nbsp;row using the _&laquo;Add a&nbsp;row&raquo;_ button or&nbsp;the _&laquo;Delete&raquo;_ link, respectively.
\ No newline at end of file
+The TreeList allows users to edit data in multiple modes. This demo shows the *"row"* edit mode. To edit any row, click Edit in it. Only one row can be in the edit state at a time.
```

Comment:
`click Edit in it` Does this refer to an Edit button? 'in it' is also unclear. 

```suggestion
The TreeList allows users to edit data in multiple modes. This demo shows the *"row"* edit mode that allows you to click a row's Edit button to edit the row. Only one row can be in the edit state at a time.
```

Reply:
The main purpose of this demo is to allow users to edit rows, not click an Edit button to edit rows. That's why can this be:
```suggestion
The TreeList allows users to edit data in multiple modes. This demo shows the *"row"* edit mode that allows you to edit rows. To do this, click a row's Edit button; only one row can be in the edit state at a time.
```
?

Reply:
I thought the purpose of this demo is to show one of the edit modes? What makes this mode diiferent? Does it enable row editing? Does it add an Edit button to rows?

The TreeList supports different data edit modes. In this demo, *"Row"* edit mode adds an Edit button to each row. Note that only one row can be in the edit state at a time.